### DISCUSSION TITLE
apt update not working for container

### DISCUSSION THREAD

**Discussion by faqall** [tteck/Proxmox#1427] [Help Needed] [answered]
I have the unifi lxc and when trying to update i receive the following:

`Ign:1 http://ftp.nl.debian.org/debian stretch InRelease
Hit:2 http://ftp.debian.org/debian bullseye InRelease               
Hit:3 http://security.debian.org bullseye-security InRelease                              
Ign:4 https://repo.mongodb.org/apt/ubuntu xenial/mongodb-org/3.6 InRelease                
Hit:5 https://repo.mongodb.org/apt/ubuntu xenial/mongodb-org/3.6 Release
Err:6 http://ftp.nl.debian.org/debian stretch Release
  404  Not Found [IP: 130.89.149.21 80]
Hit:7 http://ftp.debian.org/debian bullseye-updates InRelease
Reading package lists... Done
E: The repository 'http://ftp.nl.debian.org/debian stretch Release' no longer has a Release file.
N: Updating from such a repository can't be done securely, and is therefore disabled by default.
N: See apt-secure(8) manpage for repository creation and user configuration details.`

Can someone tell me what I should have in my /etc/apt/source.list?

**Comment by tteck** [comment, marked as answer]
https://github.com/tteck/Proxmox/issues/1399

Backup, recreate LXC, restore.

**Reply by faqall** [reply]
Thanks my search didn't net that result. I'll just follow that route. Appreciate ya!
